src/containers/ModalAuthorization.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import { Modal, Button } from "react-bootstrap";
+import styled from "styled-components";
+
+import Registration, { windowTitle } from "./Registration";
+
+const StyledModalTitle = styled(Modal.Title)`
+  text-align: center;
+  font-weight: 700;
+  font-size: 2rem;
```

Comment:
are you sure that's not possible to us bootstrap to define this styles?
how about https://getbootstrap.com/docs/4.0/utilities/text/#font-weight-and-italics

Reply:
Done